### PR TITLE
Add package manager installation options

### PR DESCRIPTION
**Pull request by Username404-59**

### CODE REVIEW

File: src/Installation/guides/package.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Arch Linux or derivatives (AUR)
+
+To install Vinegar, run the following commands (where "version" is the current version of Vinegar):
+
+```sh
+git clone https://aur.archlinux.org/vinegar.git
+cd vinegar
+makepkg -si
+pacman -U vinegar-version-x86_64.pkg.tar.zst
```
Comment (apprehensions) [inline src/Installation/guides/package.md:11]: `makepkg -i` installs the package, the pacman command here is redundant.

File: src/Installation/index.md [hunk 2]
```diff
@@ -4,3 +4,4 @@ Installing from Flatpak is recommended for users who don't want to deal with bui
 
 - [Installing from source](guides/source.md)
 - [Installing from Flatpak](guides/flatpak.md)
+- [Installing from your package manager](guides/package.md)
```
Comment (apprehensions) [inline src/Installation/index.md:7]:
```suggestion
- [Installing from package](guides/package.md)
```

**Review by apprehensions — CHANGES_REQUESTED** [review]
I suggest making the vinegar overlay seperate from your personal overlay, for example under the VinegarHQ organization.

File: src/Installation/guides/package.md (hunk 1 above)
Comment (apprehensions) [reply src/Installation/guides/package.md:11]: additionally, you can mention an aur helper being used instead.

**Comment by Username404-59** [comment]
> I suggest making the vinegar overlay seperate from your personal overlay, for example under the VinegarHQ organization.

Yeah we could do that

File: src/Installation/guides/package.md (hunk 1 above)
Comment (Username404-59) [reply src/Installation/guides/package.md:11]: > `makepkg -i` installs the package, the pacman command here is redundant.

Oh okay I thought it only built it

**Comment by Thelolguy1** [comment]
@Username404-59 https://github.com/vinegarhq/ebuild

Have fun

File: src/Installation/index.md
Comment (apprehensions) [inline src/Installation/index.md:1]: Add this to the SUMMARY.md too

**Review by apprehensions — CHANGES_REQUESTED** [review]

**Comment by apprehensions** [comment]
Lgt my

**Pull request merged**